src/status_im/contexts/profile/settings/list_items.cljs

```diff
@@ -11,7 +11,7 @@
      :blur?       true
      :action      :arrow}
     {:title       (i18n/label :t/password)
-     :on-press    not-implemented/alert
+     :on-press    #(rf/dispatch [:navigate-to-within-stack [:settings-password :settings]])
```

Comment:
I believe we should open the screen as a modal because we want to overlay a blurred screen over the settings. It might not be evident in your Figma link, but it becomes clearer in the 'Edit Profile' screen. The settings should be in the background.
[edit profile figma link
](https://www.figma.com/file/PiN0uwRgZj4L9U4N7xO7Xb/Profile-for-Mobile?type=design&node-id=4528-307713&mode=design&t=30Y7DDz7TU35sW4l-4)
@clauxx wdyt? 
```suggestion
     :on-press    #(rf/dispatch [:open-modal :settings-password])
```

Reply:
done!